auth/store.go

```diff
@@ -706,6 +706,10 @@ func (as *authStore) isOpPermitted(userName string, revision uint64, key, rangeE
 		return nil
 	}
 
+	if userName == "" {
+		return ErrUserEmpty
```

Comment:
an alternative way to do this is to check if revision == 0 and bail; no strong opinion either way